examples/toolchains/cc/WORKSPACE

```
@@ -0,0 +1,38 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+http_archive(
+    name = "rules_cc",
+    sha256 = "4dccbfd22c0def164c8f47458bd50e0c7148f3d92002cdb459c2a96a68498241",
+    urls = [
+        "https://github.com/bazelbuild/rules_cc/releases/download/0.0.1/rules_cc-0.0.1.tar.gz",
+    ]
+)
+
+local_repository(
+    name = "io_tweag_rules_nixpkgs",
+    path = "../../../"
+)
+
+load("@io_tweag_rules_nixpkgs//nixpkgs:repositories.bzl", "rules_nixpkgs_dependencies")
+rules_nixpkgs_dependencies()
+
+load("@io_tweag_rules_nixpkgs//nixpkgs:nixpkgs.bzl",
+     "nixpkgs_git_repository",
+     "nixpkgs_cc_configure"
+)
+nixpkgs_git_repository(
+    name = "nixpkgs",
+    revision = "21.11",
+    sha256 = "c77bb41cf5dd82f4718fa789d49363f512bb6fa6bc25f8d60902fe2d698ed7cc",
+)
+
+nixpkgs_cc_configure(
+        repository = "@nixpkgs",
+        fail_not_supported = False,
+)
```

As [mentioned before](https://github.com/tweag/rules_nixpkgs/pull/170#discussion_r761998319) this will name-clash with the autodetecting cc toolchain. You should set `name` explicitly.
```suggestion
nixpkgs_cc_configure(
        name = "nixpkgs_config_cc",
        repository = "@nixpkgs",
        fail_not_supported = False,
)
```